### ISSUE TITLE
Required flag, yet has default?

### ISSUE THREAD

**Issue by utdrmac** [datacharmer/dbdeployer#148]
**Describe the bug**
```
$ dbdeployer downloads add Percona-Server-8.0.28-20-Linux.x86_64.glibc2.17-minimal.tar.gz --flavor percona
Error: required flag(s) "OS", "url" not set
Usage:
  dbdeployer downloads add tarball_name [flags]

Flags:
      --OS string              Define the tarball OS (default: current OS)
```

If the flag has a default, then why is it required?

**To Reproduce**
See above

**Expected behavior**
For defaults to be used as defaults

**Comment by datacharmer** [comment]
Hi, 
Thanks for reporting this issue.
The default value is a mistake. The help should not mention it, and in fact there is no default value set for this option.
The help text will be fixed in the next release.

**Comment by datacharmer** [comment]
Fixed in version [1.65.2](https://github.com/datacharmer/dbdeployer/releases/tag/v1.65.2)
